source/includes/azure/_instances.md

```diff
@@ -55,25 +61,31 @@ Retrieve a list of all instances in a given [environment](#administration-enviro
 `numberOfCores`<br/>*int* | The number of cores provisioned for this instance. This is determined by the machine type.
 `memoryInGB`<br/>*int* | The number of memory in GB provisioned for this instance. This is determined by the machine type.
-`imagePublisher`<br/>*string* | The publisher of the instance used to create the instance
-`imageOffer`<br/>*string* | The image offer that was used to create the instance
-`imageSku`<br/>*string* | The image SKU that was used to create the instance
-`imageVersion`<br/>*string* | The image version that was used to create the instance
-`displayImage`<br/>*string* | Displayable value of the image information
+`imagePublisher`<br/>*string* | The publisher of the instance used to create the instance.
+`imageOffer`<br/>*string* | The image offer that was used to create the instance.
+`imageSku`<br/>*string* | The image SKU that was used to create the instance.
+`imageVersion`<br/>*string* | The image version that was used to create the instance.
+`displayImage`<br/>*string* | Displayable value of the image information.
 `osType`<br/>*string* | The OS type of the instance. This can be either Windows or Linux.
-`region`<br/>*string* | The region in which the instance is located
-`privateIp`<br/>*string* | The private ip address assigned to the instance
-`internalFqdn`<br/>*string* | The internal FQDN assigned to the instance
-`macAddress`<br/>*string* | The MAC address assigned to the instance
-`publicIp`<br/>*string* | The public ip address assigned to the instance
-`powerState`<br/>*string* | The status of the instance. One of the following values:    PowerState/running, PowerState/deallocating, PowerState/deallocated, PowerState/starting, PowerState/stopped, PowerState/stopping and PowerState/unknown
-`displayPowerState`<br/>*string* | The status of the instance. One of the following values: running, deallocating, deallocated, starting, stopped, stopping and unknown
-`networkInterfaces`<br/>*list* | A list of network interfaces of the instance. Contains fields: `id`, `name`, `primary`
+`region`<br/>*string* | The region in which the instance is located.
+`privateIp`<br/>*string* | The private ip address assigned to the instance.
+`internalFqdn`<br/>*string* | The internal FQDN assigned to the instance.
+`macAddress`<br/>*string* | The MAC address assigned to the instance.
+`subnetName`<br/>*string* | The name of the subnet that the instance is part of.
+`networkName`<br/>*string* | The name of the network that the instance is part of.
+`supportsPremiumIO`<br/>*boolean* | If the instance supports Premium disks. This is fixed by the machine type.
+`supportsUltraIO`<br/>*boolean* | If the instance supports Ultra disks. This is fixed by the machine type.
+`attachableDiskSlots`<br/>*integer* | The number of managed disks that can be attached to the instance. This is fixed by the machine type.
+`usedLuns`<br/>*array* | An array of all the LUNs associated with the managed disks attached to this instance.
```

Comment:
usedLuns  is a set not an array.

Reply:
in json, there is not such thing as sets, only arrays. I will keep it as is even in the java implementation uses set.